modules/core/src/main/scala/gem/math/Angle.scala

```diff
@@ -143,11 +144,12 @@ object Angle {
   def fromDoubleRadians(rad: Double): Angle =
     fromDoubleDegrees(rad.toDegrees)
 
-  /** Angle is an Abelian group, but monoid is the best we can do for now. */
-  implicit val AngleMonoid: Monoid[Angle] =
-    new Monoid[Angle] {
+  /** Angle forms a commutative group. */
+  implicit val AngleCommutativeGroup: CommutativeGroup[Angle] =
```

Comment:
Yes. It's a monoid where `|+|` is commutative and every element has an inverse, such that `a |+| -a = zero`. Also called an Abelian group. See the upper-right part of the [infographic](https://github.com/tpolecat/cats-infographic).